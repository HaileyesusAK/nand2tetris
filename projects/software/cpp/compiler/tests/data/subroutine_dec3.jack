function void no_op() {}
