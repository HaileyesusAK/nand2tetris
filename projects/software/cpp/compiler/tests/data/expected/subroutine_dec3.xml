<subroutineDec>
    <keyword> function </keyword>
    <keyword> void </keyword>
    <identifier> no_op </identifier>
    <symbol> ( </symbol>
    <parameterList>
    </parameterList>
    <symbol> ) </symbol>
    <subroutineBody>
        <symbol> { </symbol>
        <symbol> } </symbol>
    </subroutineBody>
</subroutineDec>
